src/spikeinterface/extractors/tests/test_nwbextractors.py

```diff
@@ -4,9 +4,13 @@
 from tempfile import mkdtemp
 from datetime import datetime
 
+import pytest
 import numpy as np
 from pynwb import NWBHDF5IO, NWBFile
 from pynwb.ecephys import ElectricalSeries
+from pynwb.testing.mock.file import mock_NWBFile
+from pynwb.testing.mock.device import mock_Device
+from pynwb.testing.mock.ecephys import mock_ElectricalSeries, mock_ElectrodeGroup
```

Comment:
As much as I like these testing classes, they are technically not available in older PyNWB versions - as such, you may want to specify a lower bound to PyNWB in your [testing requirements](https://github.com/catalystneuro/spikeinterface/blob/refactor_nwb_extractor/pyproject.toml#L110) to avoid the situation where the SI minimal requirements say they can support reading in an environment using an older version, but the SI testing suite itself cannot run in that environment

Reply:
Yep, this makes sense. Added that